### PR TITLE
fix(docs): add confirm resend metrics event

### PR DESCRIPTION
**Pull request by vladikoff**
[skip ci]

Seeing this here: 
image

### CODE REVIEW

**Comment by vladikoff** [comment]
@shane-tomlinson r? want to make sure this is correct.

File: docs/client-metrics.md [hunk 1]
```diff
@@ -117,6 +117,8 @@ The event stream is a log of events and the time they occurred while the user is
 
 #### confirm
 
+* resend - attempt to resend verification email
```
Comment (shane-tomlinson) [inline docs/client-metrics.md]: Can you prepend `confirm.`?

**Comment by shane-tomlinson** [comment]
with the lone nit - r+

**Comment by vladikoff** [comment]
closed via https://github.com/mozilla/fxa-content-server/commit/492f8bbfd77666e97e93a0e40fb3efa309d28fc5

**Pull request closed without merging**